features/auth/RegisterScreen.tsx

```diff
@@ -0,0 +1,193 @@
+import React, { useState } from 'react';
+import { Alert, Image, StyleSheet, Text, View, KeyboardAvoidingView, ViewStyle } from 'react-native';
+import AppInput from '../../components/AppInput'
+import AuthButton from '../../components/AuthButton';
+import auth from '@react-native-firebase/auth';
+import { Screen } from '../../utils/navigation/Screens/Screens';
+import Colors from '../../utils/const/colors/Colors';
+import { ConstantLabels, ConstantText } from '../../utils/const/constantTexts/ConstantText';
+import { SvgXml } from 'react-native-svg';
+import logoXml from '../../assets/svgxml/logo';
+import { useNavigation } from '@react-navigation/native';
+import { NativeStackNavigationProp } from '@react-navigation/native-stack';
+import { AppStackParams } from '../../utils/navigation/AppNavigation';
+import getLogo from '../../assets/svgxml/logo';
+
+const RegisterScreen = () => {
+
+    const navigation = useNavigation<NativeStackNavigationProp<AppStackParams>>();
+    const [email, setEmail] = useState('');
+    const [password, setPassword] = useState('');
+    const [confirmPassword, setConfirmPassword] = useState('');
+    const emailRegex: RegExp = /^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,}$/i;
```

Comment:
if you would have moved the emailRegex to a util like @GuyShamayMoveo suggested you wouldn't have to have it here too and you could have just reuse the function again here.